server/src/utils/auth.ts

```diff
@@ -61,5 +61,5 @@ export async function getUserInfo(
   };
 }
 
-export const verifyTwilioEmail = (email: string): boolean =>
-  email.indexOf('@twilio.com') > -1;
+export const verifyTwilioEmail = (email: string, regex: RegExp): boolean =>
```

Comment:
Should we name this `twilio` since the regex is adjustable?

Reply:
Just `twilio`? How about `verifyRegex` since the regex is adjustable?

Reply:
Yeah sorry haha. I meant to say should this name include twilio